.bazelrc

```diff
@@ -2,7 +2,6 @@
 # flag: --noremote_accept_cached
 build:darwin --remote_http_cache=https://bazel-cache.da-ext.net
 build:linux --remote_http_cache=https://bazel-cache.da-ext.net
-build:windows-ci --remote_http_cache=https://bazel-cache.da-ext.net/v1
```

Comment:
I don’t see an advantage of including it here given that caching only works in CI anyway (and even there it mostly doesn’t work :cry:) and having it duplicated creeps me out a bit.